src/app/organization/adhoc-query/create-adhoc-query/create-adhoc-query.component.html

```diff
@@ -0,0 +1,88 @@
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="adhocQueryForm" (ngSubmit)="submit()">
+
+      <mat-card-content>
+
+        <div fxLayout="column">
+
+          <mat-form-field>
+            <mat-label>Name</mat-label>
+            <input matInput required formControlName="name">
+            <mat-error *ngIf="adhocQueryForm.controls.name.hasError('required')">
+              Name is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>SQL Query</mat-label>
+            <textarea matInput required formControlName="query"></textarea>
+            <mat-error *ngIf="adhocQueryForm.controls.query.hasError('required')">
+              SQL Query is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Insert Into table</mat-label>
+            <input matInput required formControlName="tableName">
+            <mat-error *ngIf="adhocQueryForm.controls.tableName.hasError('required')">
+              Insert Into table is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Table Fields</mat-label>
+            <textarea matInput required formControlName="tableFields"></textarea>
+            <mat-error *ngIf="adhocQueryForm.controls.tableFields.hasError('required')">
+              Table Fields are <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Email</mat-label>
+            <input matInput formControlName="email">
+            <mat-error *ngIf="adhocQueryForm.controls.email.hasError('email')">
+              Email is <strong>invalid</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Report Run Frequency</mat-label>
+            <mat-select formControlName="reportRunFrequency">
+              <mat-option *ngFor="let reportRunFrequency of reportRunFrequenciesData" [value]="reportRunFrequency.id">
+                {{ reportRunFrequency.value }}
+              </mat-option>
+            </mat-select>
+          </mat-form-field>
+
+          <mat-form-field *ngIf="adhocQueryForm.controls.reportRunFrequency.value === 5">
+            <mat-label>Number of Days to run report</mat-label>
+            <input type="number" matInput required formControlName="reportRunEvery">
+            <mat-error *ngIf="adhocQueryForm.controls.reportRunEvery.hasError('required')">
+              Number of Days to run report is <strong>required</strong>
+            </mat-error>
+            <mat-error *ngIf="adhocQueryForm.controls.reportRunEvery.hasError('min')">
+              Number of Days to run report <strong>should</strong> be larger than 0
```

Comment:
```suggestion
              Number of Days to run report <strong>must be larger than 0</strong>
```

Reply:
I'll wait for the mentors review and if it's a good suggestion, I'll incorporate it in the next push.

Reply:
I'm just suggesting this according to my previous PRs, Abhay suggested this to me on that PR (I can't remember which one sorry)

Reply:
Update the message. Also, update the field label as I specified above.